DelvUI/Config/PluginConfigObject.cs

```diff
@@ -9,6 +9,8 @@ namespace DelvUI.Config
     [Serializable]
     public abstract class PluginConfigObject
     {
+        public bool portable { get; set; } = true;
```

Comment:
Please change to "Portable" to keep the naming convention within the code consistent 🙏